backend/src/lib/report-generator.js

```
@@ -123,6 +125,54 @@ async function generatePdf(audit) {
+
+async function generateEncryptedPdf(audit, password) {
+  // Genera el archivo DOCX
+  var docxReport = await generateDoc(audit);
+
+  return new Promise((resolve, reject) => {
+    libre.convert(docxReport, '.pdf', undefined, (err, pdf) => {
+      if (err) {
+        console.log(err);
+        return reject(err);
+      }
+      const os = require('os');
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)